build.py

```diff
@@ -0,0 +1,22 @@
+import urllib.request
+import zipfile
+import shutil
+import os
+print("Downloading static file zip.")
```

Comment:
Agreed, I will add comments and structure now. I will attempt to add this to some kind of flask hook.

Reply:
Sadly couldn't use submodules for this, since the govuk frontend repo requires building to be useful. Currently researching methods of doing this without hassling the developers.

Reply:
It’s been reformatted and commented for now. The best reformatting would, of course, to delete this file and replace it with a cleaver solution.